README.rst

```diff
@@ -18,135 +16,38 @@ Contributing to PEPs
-
-You can also view your PEP locally with the Sphinx-based builder,
-which will show the PEP exactly as it will appear on the preview
-of the new rendering system proposed in :pep:`676`;
-see `Rendering PEPs with Sphinx`_ for details.
-
-Finally, you can check for and fix common linting and spelling issues,
-either on-demand or automatically as you commit, with our pre-commit suite.
-See the `Contributing Guide <./CONTRIBUTING.rst>`_ for details.
-
-
-Generating HTML for Python.org
-==============================
-
-Python.org includes its own helper modules to render PEPs as HTML, with
-suitable links back to the source pages in the version control repository.
-
-These can be found `in the python.org repository
-<https://github.com/python/pythondotorg/tree/main/peps>`__.
-
-When making changes to the PEP management process that may impact python.org's
-rendering pipeline:
-
-* Clone the `python.org repository <https://github.com/python/pythondotorg/>`_.
-* Get `set up for local python.org development
-  <https://pythondotorg.readthedocs.io/install.html#manual-setup>`_.
-* Adjust ``PEP_REPO_PATH`` in ``pydotorg/settings/local.py`` to refer to your
-  local clone of the PEP repository.
-* Run ``./manage.py generate_pep_pages`` as described the `python.org docs
-  <https://pythondotorg.readthedocs.io/pep_generation.html>`__.
-
-
-Rendering PEPs with Sphinx
-==========================
-
-There is a Sphinx-rendered version of the PEPs at https://python.github.io/peps/
-(updated on every push to ``main``).
-
-**Warning:** This version is not, and should not be taken to be, a canonical
-source for PEPs whilst it remains in preview (please `report any rendering bugs
-<https://github.com/python/peps/issues/new>`_).
-The canonical source for PEPs remains https://www.python.org/dev/peps/
+intend.
 
 
-Build PEPs with Sphinx locally
-------------------------------
+Render PEPs locally
+-------------------
 
-See the `build documentation <./docs/build.rst>`__ for full step by step
-instructions on how to install, build and view the rendered PEPs with Sphinx.
-
-In summary, after installing the dependencies (preferably in a virtual
-environment) with:
-
-.. code-block:: bash
-
-    python -m pip install -r requirements.txt
-
-You can build the PEPs with sphinx by running, if your system has ``make``:
+See the `build documentation <./docs/build.rst>`__ for full
+instructions on how to render PEPs locally. In summary:
 
 .. code-block:: bash
 
-    make sphinx
+    # Install requirements
+    python -m pip install -U -r requirements.txt
 
-Otherwise, execute the ``build.py`` script directly:
+    # Render the PEPs
+    make render
 
-.. code-block:: bash
+    # or, if you don't have 'make':
+    python3 build.py
 
-    python build.py
 
-The output HTML can be found under the ``build`` directory.
+The output HTML is found under the ``build`` directory.
 
 
-``build.py`` usage
-------------------
+PEP draughting aids
+-------------------
```

Comment:
I'd never heard the word "draughting" before (my first impression was that it had something to do with heavy alcohol consumption), and had to look it up; it seems to be a BrE-specific term and at least as a non-BrE speaker/writer, it was distracting, confusing and looked out of place here. I'd suggest avoiding dialect-specific terminology, especially in such a prominent place in our README, and instead using more nuetral, internationally-understood wording.

For example, also being more consistent with the phrasing of the previous headings,

```suggestion
Linting PEPs
------------
```